webpack.config.js

```diff
@@ -0,0 +1,212 @@
+/**
+ * The contents of this file are subject to the OpenMRS Public License
+ * Version 1.0 (the "License"); you may not use this file except in
```

Comment:
Can you create a new talk post and mention about the need to have this fixed to use the latest license?